### PR TITLE
Add a Makefile

### PR DESCRIPTION
**Pull request by berendt**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2017 SAP SE or an SAP affiliate company. All rights reserved.
```
Comment (rfranzke) [inline Makefile:1]:
```suggestion
# Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved.
```

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+v1.12.0-dev
```
Comment (rfranzke) [inline VERSION:1]: What is this file used for? I guess it can be deleted

**Review by rfranzke — CHANGES_REQUESTED** [review]

File: VERSION (hunk 2 above)
Comment (berendt) [reply VERSION:1]: For the default tag of the image.

``IMAGE_TAG          := $(shell cat VERSION)``

Comment (rfranzke) [reply VERSION:1]: Hm, but why do we need it? The tag should always be the KUBERNETES_VERSION that is being used, or?

Comment (berendt) [reply VERSION:1]: The Makefile is from gardener/terraformer. There the VERSION is used as tag. KUBERNETES_VERSION makes more sense here, yes.

**Review by rfranzke — APPROVED** [review]
/lgtm
/invite @ialidzhikov

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**